.hack/update_examples.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+function show_help () {
+    echo "Usage: $0 [-h|--help | -t|--tag <tag>] (-bp|--build-push | -bpe|--build-push-example <path> | -u|--update <commit-sha> | -r|--release <version>)"
+    echo "  -h, --help                   Display help message and exit"
+    echo "  -bp, --build-push            Build the Dockerfiles of all the examples and push them to the quay.io registry"
+    echo "  -bpe, --build-push-example   Build the Dockerfile of the given example directory path, and push it to the quay.io registry"
+    echo "  -t, --tag                    To be optionally used with -bpe or -bp. Specify the tag to build with. Default tag: stable"
+    echo "  -u, --update                 Update all of the examples to depend on the specified commit SHA"
+    echo "  -r, --release                Update all of the examples to depend on the specified version"
```

Comment:
can these two be merged as one `-u`? both seem updating the files and not doing releasing jobs.